src/tools/fuzzing.h

```diff
@@ -1449,7 +1452,31 @@ class TranslateToFuzzReader {
   }
 
   Expression* makeCallRef(Type type) {
-    return makeTrivial(type); // FIXME
+    // look for a call target with the right type
+    Function* target;
```

Comment:
Should we initialize this with the current function or `nullptr`, in case `wasm.functions` is empty?

Reply:
I don't think we need to. We only exit the loop (via the break) and actually use that variable if we set it to a proper value before exiting the loop.